corehq/apps/data_interfaces/templates/data_interfaces/find_by_id.html

```diff
@@ -8,6 +9,14 @@
     {% registerurl 'list_case_exports' domain %}
     {% registerurl 'list_form_exports' domain %}
 
+    {% blocktrans %}
+        <p class="lead">Find Data by ID</p>
+        <p>
+            This page allows you to directly access a specific form data or case data page.
+            Read more on our <a href='https://confluence.dimagi.com/display/commcarepublic/Find+Data+by+ID'>help site</a>.
```

Comment:
I know we hardcode external URLs in templates all the time, but do you think it would be a good idea to move them to a constants file?  Not expecting that decision to be made in this PR, but seems like a discussion worth having.

Reply:
Hmm, what would the benefits be?